flood_forecast/evaluator.py

```diff
@@ -84,6 +84,9 @@ def evaluate_model(
             eval_log[target + "_" + evaluation_metric] = s
     return eval_log, df_train_and_test, forecast_start_idx, df_predictions
 
+def infer_on_torch_model(model, test_csv_path:str = None, datetime_start=datetime(2018, 9, 22, 0), hours_to_forecast: int = 336, decoder_params=None, dataset_params:Dict={}, num_prediction_samples:int=None):
```

Comment:
why is there 2 functions with the same name? (isn't this the same as the one on line 91?)

Reply:
Yeah good catch that was a remnant of a merge conflict.